### PR TITLE
Fix bounds ordering

### PR DESCRIPTION
**Pull request by metalgearsloth**
Requires https://github.com/space-wizards/RobustToolbox/pull/1966
https://github.com/space-wizards/RobustToolbox/pull/1958 fails tests if there's any invalid bounds.

### CODE REVIEW

**Comment by metalgearsloth** [comment]
I have no idea how clickable is failing now

File: Resources/Prototypes/Entities/Structures/Doors/Firelocks/firelock.yml [hunk 1]
```diff
@@ -113,7 +113,7 @@
       fixtures:
       - shape:
           !type:PhysShapeAabb
-          bounds: "-0.2,-0.49,-0.49,0.49" # don't want this colliding with walls or they won't close
+          bounds: "-0.49,-0.49,0.49,0.49" # don't want this colliding with walls or they won't close
```
Comment (PaulRitter) [inline Resources/Prototypes/Entities/Structures/Doors/Firelocks/firelock.yml:116]:
```suggestion
          bounds: "-0.49,-0.2,0.49,-0.49" # don't want this colliding with walls or they won't close
```

File: Resources/Prototypes/Entities/Structures/Doors/Windoors/assembly.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@
     fixtures:
     - shape:
         !type:PhysShapeAabb
-        bounds: "-0.2,-0.49,-0.49,0.49"
+        bounds: "-0.49,-0.49,0.49,-0.2"
```
Comment (PaulRitter) [inline Resources/Prototypes/Entities/Structures/Doors/Windoors/assembly.yml:18]: wait what, shouldnt it be:
```suggestion
        bounds: "-0.49,-0.2,0.49,-0.49"
```

File: Resources/Prototypes/Entities/Structures/Doors/Windoors/base.yml [hunk 3]
```diff
@@ -10,7 +10,7 @@
     fixtures:
     - shape:
         !type:PhysShapeAabb
-        bounds: "-0.2,-0.49,-0.49,0.49"
+        bounds: "-0.49,-0.49,0.49,-0.2"
```
Comment (PaulRitter) [inline Resources/Prototypes/Entities/Structures/Doors/Windoors/base.yml:13]: same question as above
```suggestion
        bounds: "-0.49,-0.2,0.49,-0.49"
```

**Review by PaulRitter — CHANGES_REQUESTED** [review]
why did i decide to review this
image

**Comment by PaulRitter** [comment]
ooop, merge during review. powerful

**Pull request merged**